.styleci.yml

```diff
@@ -1,29 +1,27 @@
+risky: true
+
 preset: symfony
 
 enabled:
   - align_double_arrow
   - concat_with_spaces
-  - short_array_syntax
   - ordered_use
   - strict
 
 disabled:
-  - blank_line_before_break
-  - blank_line_before_continue
-  - blank_line_before_throw
-  - blank_line_before_try
   - concat_without_spaces
-  - method_argument_space_strict
+  - method_argument_space
+  - native_constant_invocation_symfony
+  - native_function_invocation_symfony
+  - no_superfluous_phpdoc_tags_symfony
+  - pow_to_exponentiation
   - pre_increment
   - unalign_double_arrow
   - unalign_equals
   - yoda_style
-  - property_separation
-  - const_separation
```

Comment:
are there any more overrides we should remove with this new batch of preset changes?

Reply:
No. If there were, StyleCI would fail loudly that a fixer that was not enabled by the preset cannot be disabled. :)

Reply:
oh, no, i meant whether we should remove any overrides to get closer to the base preset, not to keep styleci from whining 😉

Reply:
Oh, I guess pre_increment could go. I don't think there'd be too many changes from that. The others will cause quite a lot of changes, however. There was also the discussion we had about the slashes on internal constants and functions. You could un-disable the two _symfony fixers listed here, or keep them disabled and enable the full versions.

Reply:
let's leave `pre_increment`. that, along with `yoda_style`, is one of those things where symphony's coding standard makes you think about things backwards in the name of (questionable) protection from yourself.

looking at overlap between laravel, symfony, and styleci's "recommended", it seems like i'm pretty clearly in the minority for `concat_without_spaces`, `method_argument_space` and `unalign_equals`. we can remove those overrides 😢

Reply:
it might be worth switching the big ones (spaces and alignment changes) in their own diff to keep things cleaner.